app/views/catalog/_reviewer_action.html.erb

```diff
@@ -0,0 +1,3 @@
+<% unless document.reviewed %>
+  <%= link_to "Review", reviewer_path(:id => document.id) %>
```

Comment:
I don't think this is the right feature - this would send the reviewer to the same page they're looking at, right?